CMakeLists.txt

```diff
@@ -163,6 +163,7 @@ if (VIVADO_DISABLE)
   message(STATUS "Disabling vivado tests.")
 else()
   if (EXISTS ${VIVADO_PATH})
+    get_filename_component(VIVADO_PATH ${VIVADO_PATH} DIRECTORY)
```

Comment:
@darthscsi I see you just did something related to CMake pointing to a file, not the directory. I think this is one way to clean that up in CMake land.